voidcrew/modules/cloning/code/computer_cloning

```diff
@@ -0,0 +1,630 @@
+#define AUTOCLONING_MINIMAL_LEVEL 3
```

Comment:
This isn't a .dm file

Reply:
will fix that no idea why it reverted back from being one